src_c/polygon.c

```diff
@@ -358,7 +467,68 @@ pg_polygon_get_vertices(pgPolygonObject *self, void *closure)
     return _pg_polygon_vertices_aslist(&self->polygon);
 }
 
+static PyObject *
+pg_polygon_get_center_x(pgPolygonObject *self, void *closure)
+{
+    return PyFloat_FromDouble(self->polygon.c_x);
+}
+
+static int
+pg_polygon_set_center_x(pgPolygonObject *self, PyObject *value, void *closure)
+{
+    double val;
+    DEL_ATTR_NOT_SUPPORTED_CHECK_NO_NAME(value);
+    if (!pg_DoubleFromObj(value, &val)) {
+        PyErr_SetString(PyExc_TypeError, "Expected a number");
+        return -1;
+    }
+    self->polygon.c_x = val;
+    return 0;
+}
+
+static PyObject *
+pg_polygon_get_center_y(pgPolygonObject *self, void *closure)
+{
+    return PyFloat_FromDouble((int)self->polygon.c_y);
```

Comment:
```suggestion
    return PyFloat_FromDouble(self->polygon.c_y);
```

Reply:
PyFloat_FromDouble required a double as input, you are passing int

Reply:
Done!